### ISSUE TITLE
Automatic Analysis wont fire if database exists but is empty

### ISSUE THREAD

**Issue by mrfearless** [ThunderCls/xAnalyzer#28]
If a database for a debuggee target exists already but is empty - say with the dbclear command (which leaves just the hash value in the database and no labels, comments, breakpoints etc approx file is 57 bytes in size) then when automatic analysis option is checked and debuggee target reaches entrypoint the auto analysis with instead fetch from the database (which is empty) and thus will not trigger the auto analysis.

If the database file is deleted then the auto analysis option will fire correctly as if it is a first time view of the debuggee target.

**Comment by ThunderCls** [comment]
Yeah, the initial approach I had regarding this was to just check for the existence of the db file and not its content, so the analysis would fire in case no db were found and the opposite if found, that's why you're getting this behavior. I can't think of another way to do it but to mess with the db content and do serveral checkings here and there or maybe just check if the db is empty or not.
